network-api/networkapi/wagtailpages/factory/youtube_regrets_page.py

```diff
@@ -35,11 +35,11 @@ def generate(seed):
     reseed(seed)
 
     try:
-        campaign_namespace = WagtailPage.objects.get(title='campaigns')
-        print('campaigns namespace exists')
+        campaign_index_page = WagtailPage.objects.get(title='campaigns')
+        print('campaign index page exists')
     except WagtailPage.DoesNotExist:
-        print('Generating a campaigns namespace')
-        campaign_namespace = MiniSiteNamespaceFactory.create(
+        print('Generating a campaign index page')
+        campaign_index_page = MiniSiteNamespaceFactory.create(
```

Comment:
same here, let's build a CampaignIndexPage

Reply:
ah def didn't catch these as I was just doing find & replace for  `campaign_namespace` to `campaign_index_page` lol 😂